### DISCUSSION TITLE
PyRolL Basic v2.1

### DISCUSSION THREAD

**Discussion by RollingTechSupport** [pyroll-project/pyroll-core#218] [Announcements]
Hello Everybody, 
Since our last meetings, the question was raised what kind of content should be included into the `pyroll-basic` meta package. 
Currently, `pyroll-basic` consists of a total of 10 packages, witch present several models.

   - pyroll-core
   - pyroll-cli
   - pyroll-report
   - pyroll-export
   - pyroll-integral-thermal
   - pyroll-lippmann-mahrenholz-force-torque
   - pyroll-wusatowski-spreading
   - pyroll-zouhar-contact
   - pyroll-freiberg-flow-stress
   - pyroll-lendl-equivalent-method

What kind of models should be added to the basic configuration?
Some mentioned elastic roll feedback, so the mill spring?

Also we have several more models like, Hitchcocks roll flattening. Also one could think off a other flow stress or spreading model. 
The sole problem would be that in papers, we would then have to refer to the version since users would need to see the difference in calculation results.

**Comment by sag-kalibreur** [comment]
I find the content of the basic package sufficient to achieve good and consistent results, everything else can be individually customized via the many plugins.

**Comment by RollingTechSupport** [comment]
Will add pyroll gripping analysis. It provides the local as well as mean bite angle.

**Reply by sag-kalibreur** [reply]
Thanks for the hint, I'll test it.

**Reply by RollingTechSupport** [reply]
We will also add `pyroll-linear-thermal-expansion` this plugin will calculate the cross-section during transports as well as the length during roll-pass according to a given  coefficient of thermal expansion. Remember that the initial profile dimensions will not be changed by this plugin. I created a issue under witch this problem will be addressed.

**Reply by RollingTechSupport** [reply]
Update done
